specification/peering/resource-manager/Microsoft.Peering/preview/2019-07-01-preview/peering.json

```diff
@@ -0,0 +1,2389 @@
+{
```

Comment:
In the future, please copy the last version as is into the first commit, so that commits can be used to diff the changes.